letsencrypt/README.rst

```diff
@@ -101,20 +100,71 @@ Further, if you force users to https, you'll need something like for nginx::
         rewrite ^   https://$server_name$request_uri? permanent;
     }
 
+A complete configuration file could look like this::
+
+    upstream odoo {
+            server 127.0.0.1:8069;
+    }
+    upstream odoochat {
+            server 127.0.0.1:8072;
+    }
+    server {
+            listen 443;
+            listen 80;
+            if ($scheme = "http") {
+                    set $redirect_https 1;
+            }
+            if ($request_uri ~ ^/.well-known/acme-challenge/) {
+                    set $redirect_https 0;
+            }
+            if ($request_uri ~ ^/pos/web/) {
+                    set $redirect_https 0;
+            }
+            if ($redirect_https) {
+                    rewrite ^   https://$server_name$request_uri? permanent;
+            }
+            server_name domain.tld www.domain.tld;
+            proxy_read_timeout 720s;
+            proxy_connect_timeout 720s;
+            proxy_send_timeout 720s;
+
+            proxy_set_header X-Forwarded-Host $host;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_set_header X-Real-IP $remote_addr;
+
+            ssl on;
```

Comment:
These should match current Mozilla recommended settings - https://wiki.mozilla.org/Security/Server_Side_TLS

There's a generator here - https://mozilla.github.io/server-side-tls/ssl-config-generator/

Might also want to add a comment with that link.